content/patterns/radio/radio-group-pattern.html

```diff
@@ -14,120 +14,113 @@
             </li>
           </ul>
           <div class="note">
-            <h4>Note</h4>
-            <p>Radio buttons in a toolbar are frequently styled in a manner that appears more like toggle buttons.
-            For an example, See the <a href="../toolbar/examples/toolbar.html">Simple Editor Toolbar Example</a>.</p>
+            <h4>注記</h4>
+            <p>ツールバー内のラジオボタンは、しばしばトグルボタンのように見えるスタイルで表示されます。例については、<a href="../toolbar/examples/toolbar.html">シンプルなエディターのツールバーの例</a>を参照してください。</p>
           </div>
         </section>
       </section>
 
       <section id="roles_states_properties">
-        <h2>WAI-ARIA Roles, States, and Properties</h2>
+        <h2>WAI-ARIA ロール、ステート、及びプロパティ</h2>
         <ul>
-          <li>The radio buttons are contained in or owned by an element with role <a href="#radiogroup" class="role-reference">radiogroup</a>.</li>
-          <li>Each radio button element has role <a href="#radio" class="role-reference">radio</a>.</li>
+          <li>ラジオボタンは、ロールが <a href="#radiogroup" class="role-reference">radiogroup</a> の要素に含まれている又は所有されています。</li>
+          <li>各ラジオボタン要素は、<a href="#radio" class="role-reference">radio</a> ロールを持ちます。</li>
           <li>
-            If a radio button is checked, the <code>radio</code> element has <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>true</code>.
-            If it is not checked, it has <a href="#aria-checked" class="state-reference">aria-checked</a> set to <code>false</code>.
+            ラジオボタンがチェックされている場合、<code>radio</code> 要素の <a href="#aria-checked" class="state-reference">aria-checked</a> は <code>true</code> に設定されます。
+            チェックされていない場合、<a href="#aria-checked" class="state-reference">aria-checked</a> は <code>false</code> に設定されます。
           </li>
-          <li>Each <code>radio</code> element is labelled by its content, has a visible label referenced by <a href="#aria-labelledby" class="property-reference">aria-labelledby</a>, or has a label specified with <a href="#aria-label" class="property-reference">aria-label</a>.</li>
-          <li>The <code>radiogroup</code> element has a visible label referenced by <a href="#aria-labelledby" class="property-reference">aria-labelledby</a> or has a label specified with <a href="#aria-label" class="property-reference">aria-label</a>.</li>
-          <li>If elements providing additional information about either the radio group or each radio button are present, those elements are referenced by the <code>radiogroup</code> element or <code>radio</code> elements with the <a href="#aria-describedby" class="property-reference">aria-describedby</a> property.</li>
+          <li>各 <code>radio</code> 要素は、そのコンテンツによってラベル付けされ、<a href="#aria-labelledby" class="property-reference">aria-labelledby</a> によって参照される可視ラベルを持つか、又は <a href="#aria-label" class="property-reference">aria-label</a> によってラベルが指定されます。</li>
```

Comment:
ここは A, B or C 構文なので、3つのいずれかのやり方が選択できることが伝わる必要があると思います。
```suggestion
          <li>各 <code>radio</code> 要素は、そのコンテンツによってラベル付けされるか、<a href="#aria-labelledby" class="property-reference">aria-labelledby</a> によって参照される可視ラベルを持つか、又は <a href="#aria-label" class="property-reference">aria-label</a> によってラベルが指定されます。</li>
```